PORTAL/jobs/_pyperformance.py

```diff
@@ -1812,10 +1812,10 @@ def pyversion(self) -> _utils.Version:
             return parsed
 
     @property
-    def build(self) -> List[str]:
+    def build(self) -> Tuple[str, ...]:
```

Comment:
This (and the related changes) are due to fixing a bug in the type annotation change.  `build` needs to be hashable, since it's part of the UploadID which is used as a key in a bunch of places.  So reverting this to a `Tuple` from `List`.